resin/settings.py

```diff
@@ -74,18 +78,18 @@ def __write_settings(self, default=None):
 
         if default:
             self._setting = Settings._setting
-        config = ConfigParser.ConfigParser()
+        config = configparser.ConfigParser()
         config.add_section(self.CONFIG_SECTION)
         for key in self._setting:
-            config.set(self.CONFIG_SECTION, key, self._setting[key])
+            config.set(self.CONFIG_SECTION, key, str(self._setting[key]))
```

Comment:
There's no code inside `__read_settings` which turns any values into a non-string value, so it seems a bit weird that `__write_settings` is specifically coercing all values to strings. For consistency, perhaps it'd make sense to initialise all the values in `self._setting` to strings upon creation, rather than deferring this step until `__write_settings` ?
i.e. change line 42 to `'image_cache_time': str(1 * 1000 * 60 * 60 * 24 * 7),` etc. and get rid of the `str` here.

Reply:
I did this because user can add a setting and it can be a non-string value but after checking the code again, I already did convert everything to string before adding so you're right, we should change this.